domains/NithinBiju.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "personal web page",
+    "repo": "https://nithinbiju007.github.io/NithinBiju/",
+    "owner": {
+        "username": "nithinbiju007",
+        "email": "nithinbijubiss@gmail.com",
+        
+    },
+    "record": {
+        "CNAME": "https://nithinbiju007.github.io/NithinBiju/"
```

Comment:
Invalid CNAME. CNAME should be a hostname only.

E.g. `dibster.com`

Not: `https://dibster.com/about`